public/components/trace_analytics/components/traces/trace_view_helpers.tsx

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { get } from 'lodash';
```

Comment:
avoid using lodash?

Reply:
Removed and refactored to work without it.

Reply:
do we have a replacement for lodash? some of them like `get` are useful and not trivial in plain js
https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore?tab=readme-ov-file#_get